toniepodcastsync.py

```diff
@@ -197,3 +206,30 @@ def __generate_chapter_title(self, ep: Episode) -> str:
     def __is_tonie_empty(self, tonie_id: str) -> bool:
         tonie = self.__tonieDict[tonie_id]
         return tonie.chaptersPresent == 0
+
+    def __adjust_volume__(self, audio_bytes: bytes, volume_adjustment: int) -> bytes:
+        audio = AudioSegment.from_file(BytesIO(audio_bytes), format="mp3")
+
+        adjusted_audio = audio + volume_adjustment
+
+        byte_io = BytesIO()
+
+        adjusted_audio.export(byte_io, format="mp3")
+
+        return byte_io.getvalue()
+
+    def __is_ffmpeg_available(self) -> bool:
+        try:
+            # Safe to use untrusted input: executable is hardcoded
+            # noqa: S603
+            executable = "ffmpeg" if platform.system().lower() != "windows" else "ffmpeg.exe"
+            subprocess.run([executable, "-version"], check=True, capture_output=True)
```

Comment:
```suggestion
            subprocess.run([executable, "-version"], check=True, capture_output=True)    # noqa: S603
```

Also think we can ignore this warning. Afaik we have to add the noqa-Tag in that specific line

Reply:
Damn, I tried that too but accidentally appended it to line 225 instead of 226 :) Now it works.